### PR TITLE
change layout file order

### PR DESCRIPTION
**Pull request by Santoshziffity**
## Purpose of this pull request

This pull request (PR) for change layout files order

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-inherit.html#theme-inherit-layout

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md [hunk 1]
```diff
@@ -130,13 +130,13 @@ You can find out what exactly code changes are required to perform this and othe
 
 The layouts processing mechanism does not involve fallback. The system collects [layout](https://glossary.magento.com/layout) files in the following order:
 
-1. All modules layout files in sequence defined in `app/etc/config.php` respecting the [component load order]({{ page.baseurl }}/extension-dev-guide/build/module-load-order.html). For each module:
+1. Current theme layouts: `<theme_dir>/<Vendor>_<Module>/layout/`
 
-   *  Layout files for the `base` area: `<module_dir>/view/base/layout/`
-   *  Layout files for the `frontend` area: `<module_dir>/view/frontend/layout/`
+1. Ancestor themes layouts, starting from the most distant ancestor, recursively until a theme with no parent is reached: `<parent_theme_dir>/<Vendor>_<Module>/layout/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:135]:
```suggestion
1. Ancestor theme layouts, starting from the most distant ancestor, recursively until a theme with no parent is reached: `<parent_theme_dir>/<Vendor>_<Module>/layout/`
```

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md [hunk 2]
```diff
@@ -130,13 +130,13 @@ You can find out what exactly code changes are required to perform this and othe
...
 
-1. Ancestor theme layouts, starting from the most distant ancestor, recursively until a theme with no child is reached: `<parent_theme_dir>/<Vendor>_<Module>/layout/`
-1. Current theme layouts: `<theme_dir>/<Vendor>_<Module>/layout/`
+1. All modules layout files in sequence defined in app/etc/config.php respecting the component load order. For each module:
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:137]:
```suggestion
1. All module layout files in sequence, defined in `app/etc/config.php` respecting the component load order. For each module:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Few small grammar corrections

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Comment by keharper** [comment]
@Santoshziffity This topic was changed two weeks ago in https://github.com/magento/devdocs/pull/8599 to remove the references to OrangeCompany. It looks like they might have missed an item. Please revert the examples to "SampleCompany" and update other references in this top to OrangeCompany or OrangeCo

I don't understand why you changed the example so that the copyright page is removed from the footer instead of a report bugs page. The original example seems much more likely.

**Comment by BarnyShergold** [comment]
@Santoshziffity - I am about to change all occurrences of OrangeCo to ExampleCorp (as this is used elsewhere) . So for consistency, please can you change all occurrences of OrangeCo AND SampleCompany in this file to ExampleCorp and we'll review. This will avoid any conflicts with the PR I am about to make. Thanks

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**